### PR TITLE
Fix log message

### PR DESCRIPTION
**Pull request by rafahop**
### What

Fix log messages: 
 - the creation hook wrongly mentions delete
 - the create endpoint wrongly mentions get

### How to review

Sense check

### Who can review

Anyone

### CODE REVIEW

File: zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java [hunk 1]
```diff
@@ -337,15 +337,15 @@ private boolean execCreationHook(com.github.onsdigital.zebedee.content.page.base
                                      Collection collection, Session session) {
         info().data("path", uri).data("collection_id", collection.getDescription().getId()).data("user",
                 session.getEmail())
-                .log("page delete endpoint: executing PageCreationHook");
+                .log("page create endpoint: executing PageCreationHook");
```
Comment (goofballLogic) [inline zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java:340]: how do you know which endpoint is calling this? maybe the "page create endpoint" bit of the string needs to be passed in?

Comment (rafahop) [reply zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java:340]: How do I know? 🤔  Just follow the trace and you'll see it is called from the create page endpoint

Comment (goofballLogic) [reply zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java:340]: shall i remove this comment and let someone else review instead?

**Review by mike-kelly-methods — APPROVED** [review]
👍

**Pull request merged**